LEAME

```diff
@@ -1,207 +1,179 @@
-
-Una vez adjuntado los datos secretos como se vió antes podemos enviar el archivo 
-imagen.jpg a la persona que debería recibir el mensaje secreto. El receptor debe 
-usar steghide de la siguiente manera:
-
-  $ steghide extract -sf imagen.jpg
- 
- Ingrese salvoconducto:
- 
- los datos extraídos se grabaron en "secreto.txt".
- 
-Si el salvoconducto declarado es el correcto, el contenido del archivo original 
-secreto.txt se extraerá del archivo stego imagen.jpg y se grabará en 
-el directorio actual.
-
-Si hemos recibido un archivo que contiene datos adjuntos y quisiéramos tener 
-más información sobre el mismo antes de extraerlo, usaremos el comando info:
-
-  $ steghide info archivo_recibido.wav
- 
- "archivo_recibido.wav":
- 
-   formato: wave audio, PCM encoding
-   
-   capacidad: 3.5 KB
-  ¿
-  Intento extraer información sobre los datos adjuntos? (y/n) y
- 
- Ingrese salvoconducto:
- 
-   archivo adjunto "secreto.txt":
-   
-     tamaño: 1.6 KB
-     
-     encriptado: rijndael-128, cbc
-     
-     comprimido: si
-     
-Luego de emitir algunos datos generales sobre el archivo stego (formato, capacidad) se nos preguntará 
-si queremos que steghide obtenga información sobre los datos adjuntos. Si respondemos con 
-yes tendremos que declarar un salvoconducto. Steghide entonces intentará extraer los datos adjuntos
-con ese salvoconducto y - si tiene 'éxito - emitirá un informe sobre el mismo.
-
-Contacto:
-=========
-
-La Sede :
----------
-La versión más reciente de steghide e información adicional y la documentación 
-pueden obtenerse de la sede WWW steghide en:
-http://steghide.sourceforge.net/
-
-Listas de Correo :
-------------------
-Si encontramos algún fallo o si tenemos preguntas, comentarios, sugerencias, etc.
-por favor, enviarme un email a la lista de correo de desarrollo:
-steghide-devel@lists.sourceforge.net
-Para recibir emails enviados a esta lista, podemos suscribirnos en :
-http://lists.sourceforge.net/lists/listinfo/steghide-devel
-
-Si queremos mantenernos informados de cuando aparece una nueva versión de steghide 
-por favor, suscribirse a la lista de correo de anuncios de steghide en:
-http://lists.sourceforge.net/lists/listinfo/steghide-announce
-
-Acceso a CVS anónimo :
-----------------------
-
-Podemos acceder al código de desarrollo más reciente mediante cvs anónimo. Solo hay
-que escribir las siguientes líneas:
-
-$ cvs -d:pserver:anonymous@cvs.steghide.sourceforge.net:/cvsroot/steghide login
-CVS password:  [ Acá pulsar RETURN ]
-
-$ cvs -z3 -d:pserver:anonymous@cvs.steghide.sourceforge.net:/cvsroot/steghide co steghide
-
-También podemos navegar por el repositorio de cvs en WWW:
-http://cvs.sourceforge.net/cgi-bin/viewcvs.cgi/steghide/
-
-Autor :
--------
-Pueden contactarme (Stefan Hetzl) por e-mail: shetzl@chello.at
-
-
-Traducción al castellano :
---------------------------
-versión 1 - Alberto Adrián Schiano (chanio@users.sourceforge.net)
-
+Introducci�n :

```

Comment:
Can you check the encoding here?

Reply:
Not only checked but also fixed. Somehow this turned into ISO-8859-1. Reverted it back to UTF-8.